pom.xml

```diff
@@ -198,6 +198,7 @@
         <checker.qual.version>3.10.0</checker.qual.version>
         <awaitility.version>4.2.0</awaitility.version>
         <jwt.version>0.10.7</jwt.version>
+        <httpclient.version>4.5.13</httpclient.version>
```

Comment:
Please not define the dependency that connector needed in main pom, you should define it in your connector own pom.